Source/CLI/Main.cpp

```diff
@@ -127,12 +127,15 @@ bool parse_info::ParseFile_Input(input_base& SingleFile, bool OverrideCheckPaddi
         if (strstr(Global.Errors.ErrorMessage(), "becoming too big"))
         {
             Global.ProgressIndicator_Stop();
-            cerr << "Error: the reversibility file is becoming unexpectedly big.\n"
+            cerr << "Error: the reversibility file is becoming big.\n"
                     "       FFmpeg, used for muxing the output, has some issues with big\n"
                     "       attachments, and such big reversibility file is not expected\n"
                     "       with such compression, we prefer to be safe and we reject the\n"
                     "       compression.\n"
-                    "       Please contact info@mediaarea.net if you want support of such input." << endl;
+                    "       Use \"--version 2\" option for supporting\n"

```

Comment:
Adding a not here as a reminder that "--output-version 2" is current snapshot flag, in case this remains the error code will need updating. Thanks for the fix!

Reply:
Just noticed this as well, looks like it's still present actually.